kuwala/pipelines/population-density/src/Downloader.py

```diff
@@ -62,14 +62,18 @@ def download_files(dataset: dict, args) -> [str]:
                 file_path_without_ext = file_path.replace('.CSV', '')
                 os.rename(file_path, file_path_without_ext)
 
+                date=r['updated'].replace('-','_')
                 with zipfile.ZipFile(file_path_without_ext, 'r') as zip_ref:
                     zip_ref.extractall(dir_path_type)
-                file_path_with_update_date=file_path_without_ext.split('_csv')[0]+'_'+r['updated']+'.csv'
+                file_path_with_update_date=file_path_without_ext.split('/')
+                file_path_with_update_date[-1]=date+'_'+file_path_with_update_date[-1]
```

Comment:
Is it better for sorting if we give the filename like this? 🤔
```suggestion
                file_path_with_update_date[-1]=file_path_with_update_date[-1]+'_'+date
```